src/gluonts/block/encoder.py

```diff
@@ -224,13 +224,15 @@ def hybrid_forward(
             shape (batch_size, sequence_length, num_dynamic_features)
         """
 
-        if self.use_covariates:
-            inputs = Seq2SeqEncoder._assemble_inputs(
+        if self.use_dynamic_feat and self.use_static_feat:
+            inputs = self._assemble_inputs(
                 F,
                 target=target,
                 static_features=static_features,
                 dynamic_features=dynamic_features,
             )
+        elif self.use_dynamic_feat:
+            inputs = F.concat(target, dynamic_features, dim=2)  # (N, T, C)
         else:
             inputs = target
```

Comment:
Isn't this already handling the possibility of disabling the use of dynamic features?